core/domain/exp_jobs_one_off.py

```diff
@@ -89,6 +89,28 @@ def reduce(exp_id, committer_id_list):
         exp_summary_model.put()
 
 
+class ExplorationContributorsSummary(jobs.BaseMapReduceJobManager):
+    """One-off job that computes the number of commits
+    done by contributors for each Exploration
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        summary = exp_services.get_exploration_summary_by_id(item.id)
+        summary.contributor_summary = exp_services.compute_exploration_contributors_summary(item.id)
```

Comment:
This is > 80 chars; do this instead:


        summary.contributor_summary = (
            exp_services.compute_exploration_contributors_summary(item.id))